Expecta/Expecta.h

```diff
@@ -12,3 +12,4 @@ FOUNDATION_EXPORT const unsigned char ExpectaVersionString[];
 
 // Enable shorthand by default
 #define expect(...) EXP_expect((__VA_ARGS__))
+#define expect_fail(...) EXP_expect_fail((__VA_ARGS__))
```

Comment:
hmm, didn't think of that. `failure` then? Swift has `fatalError` which seems very much in spirit…